packages/jest-types/src/Config.ts

```diff
@@ -223,6 +224,25 @@ type NotifyMode =
   | 'success-change'
   | 'failure-change';
 
+type DisplayNameColor =
```

Comment:
Would be good to extract these from `chalk` instead of hard coding them

Reply:
Oh I was hard coding this because of what I mentioned about not being sure if we should curate the colors that we should allow or if we should allow everything. Hence the early PR 😄

Reply:
I've taken a look at the types provided by chalk and they don't expose the colors. https://github.com/chalk/chalk/blob/master/index.d.ts#L231

Reply:
Just as an FYI, opened up this PR. Hopefully this gets merged and we won't have to hardcode colors here. https://github.com/chalk/chalk/pull/336